acceptance/tests/custom_facts/error_in_custom_file_does_not_affect_cli.rb

```diff
@@ -0,0 +1,100 @@
+test_name 'Facter cli works when there is an error inside a custom fact file' do
+  tag 'risk:high'
+
+  first_file_content = <<-EOM
+    Facter.add(:custom_fact_1) do
+      setcode do
+        'custom_fact_1_value'
+      end
+    end
+     
+    # some error
+    nill.size
+     
+    Facter.add(:custom_fact_2) do
+      setcode do
+        'custom_fact_2_value'
+      end
+    end
+     
+    Facter.add(:custom_fact_3) do
+      setcode do
+        'custom_fact_3_value'
+      end
+    end
+  EOM
+
+  second_file_content = <<~EOM
+    Facter.add(:custom_fact_4) do
+      setcode do
+        'custom_fact_4_value'
+      end
+    end
+  EOM
+
+  def create_custom_fact_file(file_name, file_content, fact_dir, agent)
+    fact_file = File.join(fact_dir, file_name)
+    create_remote_file(agent, fact_file, file_content)
+  end
+
+  agents.each do |agent|
+    fact_dir = agent.tmpdir('facter')
```

Comment:
Maybe we can rename this to `custom_facts` and `custom_facts_dir`

Reply:
done